appserver/tests/embedded/maven-plugin/standalonewar/src/main/java/test/ServletTest3.java

```diff
@@ -17,31 +17,8 @@
 package test;
 
 import java.io.*;
-import java.net.*;
```

Comment:
Could you yet replace those wildcard imports? Not sure how Netbeans, Eclipse can to that automatically.